server/metrics/metrics.go

```diff
@@ -183,6 +185,15 @@ func NewMetrics(info InstanceInfo) Metrics {
 	}, []string{"action", "source", "is_direct", "discarded_reason"})
 	m.registry.MustRegister(m.filesCount)
 
+	m.messagesConfirmedCount = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace:   MetricsNamespace,
+		Subsystem:   MetricsSubsystemEvents,
+		Name:        "messages_confirmed",
```

Comment:
Can we append `_total` as a suffix? I missed this in the original design document.